### PR TITLE
provider/google: Added url map resource + tests & documentation

### PR DESCRIPTION
**Pull request by lwander**

### CODE REVIEW

File: builtin/providers/google/resource_compute_url_map.go [hunk 1]
```diff
@@ -0,0 +1,650 @@
...
+		urlMap.Tests[i] = createUrlMapTest(v)
+	}
+
+	op, err := config.clientCompute.UrlMaps.Insert(config.Project, urlMap).Do()
+
+	if err != nil {
+		return fmt.Errorf("Error, failed to insert Url Map %s: %s", name, err)
+	}
+
+	err = computeOperationWaitGlobal(config, op, "Insert Url Map");
```
Comment (sparkprime) [inline builtin/providers/google/resource_compute_url_map.go]: semicolon on the end of the line

**Comment by catsby** [comment]
Hey @lwander, `go fmt ./...` picked up a few things as well, can we run that on this PR? Thanks!

File: website/source/docs/providers/google/r/compute_url_map.html.markdown [hunk 2]
```diff
@@ -0,0 +1,136 @@
+---
+layout: "google"
+page_title: "Google: google_compute_url_map"
+sidebar_current: "docs-google-resource-url_map"
+description: |-
+  Manages a VM instance resource within GCE.
+---
+
+# google\_compute\_instance
```
Comment (sparkprime) [inline website/source/docs/providers/google/r/compute_url_map.html.markdown]: title is wrong

File: website/source/docs/providers/google/r/compute_url_map.html.markdown [hunk 3]
```diff
@@ -0,0 +1,136 @@
+---
+layout: "google"
+page_title: "Google: google_compute_url_map"
+sidebar_current: "docs-google-resource-url_map"
+description: |-
+  Manages a VM instance resource within GCE.
```
Comment (sparkprime) [inline website/source/docs/providers/google/r/compute_url_map.html.markdown]: description is wrong

**Comment by lwander** [comment]
@catsby, sure thing! I'll squash once Dave finishes looking over this PR.

**Pull request merged**